diff --git a/app/models/video-view.js b/app/models/video-view.js
new file mode 100644
index 0000000..6f170ce
--- /dev/null
+++ b/app/models/video-view.js
@@ -0,0 +1,9 @@
+import DS from 'ember-data';
+
+const { attr, belongsTo } = DS;
+
+export default DS.Model.extend({
+  finished: attr('boolean'),
+  user: belongsTo(),
+  clip: belongsTo()
+});
diff --git a/mirage/scenarios/default.js b/mirage/scenarios/default.js
index 5e345ad..5b839bd 100644
--- a/mirage/scenarios/default.js
+++ b/mirage/scenarios/default.js
@@ -11,8 +11,8 @@ import userStates from './user-states';
 
 // Choose one
 const state =
-  'anonymous'
-  // 'unsubscribed'
+  // 'anonymous'
+  'unsubscribed'
   // 'subscribed'
 ;
 
