diff --git a/app/pods/topics/topic/index/route.js b/app/pods/topics/topic/index/route.js
index 6c74252..a1e61c7 100644
--- a/app/pods/topics/topic/index/route.js
+++ b/app/pods/topics/topic/index/route.js
@@ -1,4 +1,11 @@
 import Route from '@ember/routing/route';
 
 export default Route.extend({
+
+  model() {
+    let series = this.modelFor('topics.topic');
+
+    return series.reloadWith('clips.encodes');
+  }
+
 });
diff --git a/app/pods/topics/topic/route.js b/app/pods/topics/topic/route.js
index b4404c1..83f36f4 100644
--- a/app/pods/topics/topic/route.js
+++ b/app/pods/topics/topic/route.js
@@ -8,11 +8,8 @@ export default Route.extend({
     let filter = { slug };
 
     return this.store
-      .loadAll('series', {
-        filter,
-        // include: 'clips.encodes'
-      })
-      .then(posts => posts.get('firstObject'));
+      .loadAll('series', { filter })
+      .then(series => series.get('firstObject'));
   },
 
   afterModel(topic) {
diff --git a/mirage/scenarios/default.js b/mirage/scenarios/default.js
index 5b839bd..b36a743 100644
--- a/mirage/scenarios/default.js
+++ b/mirage/scenarios/default.js
@@ -12,8 +12,8 @@ import userStates from './user-states';
 // Choose one
 const state =
   // 'anonymous'
-  'unsubscribed'
-  // 'subscribed'
+  // 'unsubscribed'
+  'subscribed'
 ;
 
 export default function(server) {
diff --git a/mirage/scenarios/user-states.js b/mirage/scenarios/user-states.js
index f4c3684..223f591 100644
--- a/mirage/scenarios/user-states.js
+++ b/mirage/scenarios/user-states.js
@@ -53,6 +53,17 @@ export default {
       });
     }
 
+    // Create a video view
+    let mirageSeries = server.schema.series
+      .where({ slug: 'mirage-tips-and-tricks' })
+      .models[0];
+    server.create('video-view', {
+      user,
+      clip: mirageSeries.clips.models[0],
+      finished: true
+    });
+
+    // Create some receipts
     server.create('receipt-email', {
       team,
       email: 'accounting@embermap.com'
