diff --git a/app/pods/components/video-list/template.hbs b/app/pods/components/video-list/template.hbs
index e18204f..760427a 100644
--- a/app/pods/components/video-list/template.hbs
+++ b/app/pods/components/video-list/template.hbs
@@ -1,5 +1,5 @@
 <ul class='list-reset'>
-  {{#each (sort-by 'position' videos) as |video|}}
+  {{#each (sort-by 'position' videos) as |video i|}}
     <li class={{if (eq activeVideo video) 'bg-em-orange -mx-3 px-3 lg:rounded'}}>
       {{#ui-link 'topics.topic.video' video
         style=(concat 'block tall ' (if (eq activeVideo video) 'white'))
@@ -15,40 +15,54 @@
             {{video.title}}
           </div>
 
-          <div class="w-64 xs:w-128 mt-px pt-px text-right pr-2 text-8 ml-auto mr-0">
+          {{!-- <div class="mt-px pt-px text-right pr-2 text-8 ml-auto mr-0">
             {{#if video.isComingSoon}}
               <em data-test-id="coming-soon-video">Coming soon</em>
             {{else}}
-              <div>
-                <span class='m-0 p-1 {{if (eq activeVideo video) "text-white" "text-grey-dark"}}'
-                  data-tether-id={{concat 'viewed-video-' video.id}}
-                >
-                  {{fa-icon 'check'}}
-                </span>
-                {{#ui-tooltip attachTo=(concat "[data-tether-id='viewed-video-" video.id "']")}}
-                  You've watched this video.
-                {{/ui-tooltip}}
-
-                {{#if (and
-                  video.isPremium
-                  (not currentSession.currentUser.isSubscribed)
-                )}}
-                  <span class='m-0 p-1 {{if (eq activeVideo video) "text-white" "text-grey-dark"}}'
-                    data-test-id="locked-video"
-                    data-tether-id={{concat 'premium-video-' video.id}}
-                  >
-                    {{fa-icon 'lock'}}
-                  </span>
-                  {{#ui-tooltip attachTo=(concat "[data-tether-id='premium-video-" video.id "']")}}
-                    This is a premium video.
-                  {{/ui-tooltip}}
-                {{/if}}
-
-                {{time-format video.duration}}
-              </div>
+              {{time-format video.duration}}
+            {{/if}}
+          </div> --}}
+        </div>
+        <div class="flex text-8">
+          <div class="w-32 mr-1"></div>
 
+          <div class="pr-2">
+            {{#if video.isComingSoon}}
+              <em data-test-id="coming-soon-video">Coming soon</em>
+            {{else}}
+              {{time-format video.duration}}
             {{/if}}
           </div>
+
+          {{#if (lt i 3)}}
+            <span
+              class='
+                m-0 mr-2
+                {{if (eq activeVideo video) "text-white" "text-green"}}
+              '
+              data-tether-id={{concat 'viewed-video-' video.id}}
+            >
+              {{fa-icon 'check'}} Played
+            </span>
+            {{#ui-tooltip attachTo=(concat "[data-tether-id='viewed-video-" video.id "']")}}
+              You've watched this video.
+            {{/ui-tooltip}}
+          {{/if}}
+
+          {{#if (and
+            video.isPremium
+            (not currentSession.currentUser.isSubscribed)
+          )}}
+            <span class='m-0 mr-2 {{if (eq activeVideo video) "text-white" "text-grey-dark"}}'
+              data-test-id="locked-video"
+              data-tether-id={{concat 'premium-video-' video.id}}
+            >
+              {{fa-icon 'lock'}} Premium
+            </span>
+            {{#ui-tooltip attachTo=(concat "[data-tether-id='premium-video-" video.id "']")}}
+              This is a premium video.
+            {{/ui-tooltip}}
+          {{/if}}
         </div>
       {{/ui-link}}
     </li>
diff --git a/mirage/factories/series.js b/mirage/factories/series.js
index eedafdd..ccc6804 100644
--- a/mirage/factories/series.js
+++ b/mirage/factories/series.js
@@ -27,7 +27,7 @@ export default Factory.extend({
     afterCreate(series, server) {
       server.create('clip', 'withEncodes', { series, title: 'Intro' });
       server.create('clip', 'withEncodes', { series, viewingRequirement: 'subscription', title: 'Paid 1' });
-      server.create('clip', 'withEncodes', { series, viewingRequirement: 'subscription', title: 'Paid 2' });
+      server.create('clip', 'withEncodes', { series, viewingRequirement: 'subscription', title: 'Paid 2, the one with the really, really, really long name' });
       server.create('clip', 'withEncodes', { series, viewingRequirement: 'account', title: 'Email wall 1' });
       server.create('clip', 'withEncodes', { series, viewingRequirement: 'subscription', comingSoon: true, title: 'Coming soon' });
     }
