diff --git a/tests/acceptance/video-test.js b/tests/acceptance/video-test.js
index f3db812..09d7261 100644
--- a/tests/acceptance/video-test.js
+++ b/tests/acceptance/video-test.js
@@ -120,4 +120,23 @@ module('Acceptance | Video', function(hooks) {
     assert.equal(currentURL(), `/video/this-doesnt-exist`);
     assert.equal(currentRouteName(), "404");
   });
+
+  test(`I should see a played label if I've watched a video`, async function(assert) {
+    // Assemble
+    let currentUser = loginUser();
+    this.server.create('video-view', {
+      user: currentUser,
+      clip: this.freeClip
+    });
+
+    // Act
+    await visit(`/topics/${this.series.slug}/${this.freeClip.slug}`);
+
+    // Assert
+    await this.inspect();
+    // assert.elementExists(testId("series-description"));
+  });
+
+  // test(`I cannot see other users video views`, async function(assert) {
+  // })
 });
